src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding: 0 50px;
+
+  &__logo {
+    align-self: center;
+    justify-content: center;
+
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    font-size: 12px;
+    font-weight: 500;
+
+    display: flex;
+    text-align: center;
+    align-items: center;
+
+    text-transform: uppercase;
+
+    color: $color-main;
+
+    height: 60px;
+
+    &:hover {
+      color: $color-blue-accent;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $color-blue-accent;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+
+  bottom: 0;
+  left: 0;
+
+  height: 4px;
+  width: 100%;
+
+  background-color: $color-blue-accent;
+
+  border-radius: 8px;
+}
```

Comment:
Are you sure this needs to be a separate block? Maybe it makes sense to make this modifier?